services/src/utils.ts

```diff
@@ -12,19 +12,18 @@ const emailValidator = (email: string): boolean =>
 export const changeTaskStatus = (status: TaskStatus): TaskStatus =>
   status === TaskStatus.Completed ? TaskStatus.Uncompleted : TaskStatus.Completed;
 
-const auth = (context: Context): string => {
-  const Authorization = context.request.get('Authorization');
+const auth = (Authorization: string): string => {
```

Comment:
Could you remove  below line. I think it's a leftover:
https://github.com/BuildIt-Poland/buddy-app/blob/1362944cf745c4ba09546733989311b00b02555a/services/src/utils.ts#L1

Reply:
Outdated